### PR TITLE
SP-335 - Migrate to Contentful

### PR DESCRIPTION
**Pull request by fakhergh**

### CODE REVIEW

**Comment by bouthouri** [comment]
Task linked: [SP-335 Migrate to Contentful](https://app.clickup.com/t/2427294/SP-335)

File: src/components/ArticleContent/ArticleContent.js [hunk 1]
```diff
@@ -1,62 +1,97 @@
...
+              />
             </a>
-            <a href={social_media.pinterest.link} target="_blank">
-              <img src={social_media.pinterest.icon} /> Enregistrer
+            <a href="#" target="_blank">
+              <img
+                src={
+                  "https://www.sorpetaler.de/wp-content/themes/hubert/assets/images/icon_pinterest_share.svg"
+                }
+              />{" "}
```
Comment (fakhergh) [inline src/components/ArticleContent/ArticleContent.js:76]: @Gatreek92 this is Prettier auto-generated space, we don't need it.

**Review by fakhergh — COMMENTED** [review]
@Gatreek92 in the ArticleContent.js file line 132, you need to add the @ to the media query. I was not able to comment it during my review, then I putted it here as a note.

File: src/components/RelatedPosts/RelatedPosts.js [hunk 2]
```diff
@@ -1,23 +1,55 @@
...
-          <ArticlePreview
-            key={index}
-            title={post.title}
-            description={post.excerpt}
-            img={post.img}
-            link={post.link}
-            category={post.category}
-          />
-        ))}
+        <StaticQuery
```
[the remark this answers is not in the document]
Comment (Gatreek92) [reply src/components/RelatedPosts/RelatedPosts.js:32]: @fakhergh I couldn't do it, I think static query does not accept variables.
let me know if you know how to do it

File: src/components/ArticleContent/ArticleContent.js [hunk 3]
```diff
@@ -1,62 +1,91 @@
...
+          <a href="#">{title}</a>
         </ArticleTitle>
         <Introduction>{introduction}</Introduction>
         <hr />
         <Author>
           <div>
             Von{" "}
             <a href={author.link} rel="author">
-              {author.name}
+              SANDRA
```
Comment (bouthouri) [inline src/components/ArticleContent/ArticleContent.js:45]: This is hardcoded

**Review by bouthouri — APPROVED** [review]

**Pull request merged**